packages/dataviews/src/dataform-controls/index.tsx

```diff
@@ -51,7 +53,7 @@ export function getControl< Item >(
 		return getControlByType( field.Edit );
 	}
 
-	if ( field.elements ) {
```

Comment:
This is okay for now, it's how it's already working. I think a better approach would be to actually remove this from here and make every field definition absorb this (handle how they want to use or not use the `elements` information). Leaving this as feedback in case you are available to follow up on this.